example/spec/parallel/cnmWorkflow/LambdaEventSourceSpec.js

```diff
@@ -70,7 +70,7 @@ describe('When adding multiple rules that share a kinesis event stream', () => {
     // clean up stack state added by test
     console.log(`\nCleaning up stack & deleting test stream '${streamName}'`);
     await deleteRules(testConfig.stackName, testConfig.bucket, rulesToDelete, ruleSuffix);
-    await deleteExecution({ prefix: testConfig.stackName, executionArn: executionArn });
+    await waitForExecutionAndDelete(testConfig.stackName, executionArn, 'completed');
     await Promise.all([
```

Comment:
Fix error due to the execution not deleted before deleting collection.
-------------------
>> An error was thrown in an afterAll
--
20-Jan-2023 14:23:44 | error properties: Object({ statusCode: 400, apiMessage: '{"statusCode":400,"error":"Bad Request","message":"delete from \"collections\" where \"name\" = $1 and \"version\" = $2 - update or delete on table \"collections\" violates foreign key constraint \"executions_collection_cumulus_id_foreign\" on table \"executions\"","length":335,"name":"error","severity":"ERROR","code":"23503","detail":"Key (cumulus_id)=(15) is still referenced from table \"executions\".","schema":"public","table":"executions","constraint":"executions_collection_cumulus_id_foreign","file":"ri_triggers.c","line":"3283","routine":"ri_ReportViolation"}' })
20-Jan-2023 14:23:44 | Error: /collections/L2_HR_PIXC_test-jl-rds-tf-LambdaEventSourceTest-1674242060667/000 returned 400: {"statusCode":400,"error":"Bad Request","message":"delete from \"collections\" where \"name\" = $1 and \"version\" = $2 - update or delete on table \"collections\" violates foreign key constraint \"executions_collection_cumulus_id_foreign\" on table \"executions\"","length":335,"name":"error","severity":"ERROR","code":"23503","detail":"Key (cumulus_id)=(15) is still referenced from table \"executions\".","schema":"public","table":"executions","constraint":"executions_collection_cumulus_id_foreign","file":"ri_triggers.c","line":"3283","routine":"ri_ReportViolation"}
20-Jan-2023 14:23:44 | at retries /uncached/cumulus/packages/api-client/cumulusApiClient.js:43:19
20-Jan-2023 14:23:44 | at processTicksAndRejections internal/process/task_queues.js:95:5
20-Jan-2023 14:23:44 | at async invokeApi /uncached/cumulus/packages/api-client/cumulusApiClient.js:29:12
20-Jan-2023 14:23:44 | at async Object.deleteCollection /uncached/cumulus/packages/api-client/collections.js:69:12
20-Jan-2023 14:23:44 | at async Promise.all index 0
20-Jan-2023 14:23:44 | at async deleteCollections /uncached/cumulus/packages/integration-tests/index.js:237:10
20-Jan-2023 14:23:44 | at async Promise.all index 1
20-Jan-2023 14:23:44 | at async cleanUp /uncached/cumulus/example/spec/parallel/cnmWorkflow/LambdaEventSourceSpec.js:74:5
20-Jan-2023 14:23:44 | at async UserContext.<anonymous> /uncached/cumulus/example/spec/parallel/cnmWorkflow/LambdaEventSourceSpec.js:113:5